python/pyfunc-server/pyfuncserver/test_model.py

```diff
@@ -26,25 +26,34 @@
 from prometheus_client import Counter, Gauge
 
 from pyfuncserver import PyFuncModel
+from merlin.model import PYFUNC_EXTRA_ARGS_KEY, PYFUNC_MODEL_INPUT_KEY
 
-
-class ModelImpl(mlflow.pyfunc.PythonModel):
+class NewModelImpl(mlflow.pyfunc.PythonModel):
```

Comment:
Can we add test for new model separately? and ensure the old model still satisfy this.